Snakefile

```diff
@@ -4,22 +4,28 @@ from socket import getfqdn
 from getpass import getuser
 from snakemake.logging import logger
 import sys
+from shutil import which
 
 #
 # Verify that required versions of dependencies are installed.
 #
 MIN_AUGUR_VERSION = "7.0.2"
-
 try:
     from augur.__version__ import __version__ as augur_version
 except ModuleNotFoundError:
     logger.error("ERROR: Could not find augur. Follow installation instructions at https://nextstrain.org/docs/ and try again.")
     sys.exit(1)
-
 if version.parse(augur_version) < version.parse(MIN_AUGUR_VERSION):
```

Comment:
This is a minor pedantic point, but I'd prefer to keep this whitespace for clarity. Since `try`/`except` can also end with an `else`, it's a little strange to see an `if` immediately follow in that same block.

Reply:
👍 Removed my deletions